core/common/src/ElementProps.ts

```diff
@@ -408,7 +408,10 @@ export interface ElementAspectProps extends EntityProps {
  * @extensions
  */
 export interface ExternalSourceAspectProps extends ElementAspectProps {
-  /** An element that scopes the combination of `kind` and `identifier` to uniquely identify the object from the external source. */
+  /** An element that scopes the combination of `kind` and `identifier` to uniquely identify the object from the external source.
+   * @note Warning: in a future major release the `scope` property will be optional, since the scope is intended to be potentially invalid.
```

Comment:
no... originally I had it saying here `iTwin.js@5` because that was when I intended to do it. But I also can't `@deprecate` and I couldn't find other JSDoc tags that worked for this. would you recommend I just add an issue and put it on the 5.x milestone or something?

Reply:
will do. the transformation service is waiting for this PR so I didn't want to have the debate of when to make this change, hence why I removed the iTwin.js@5 reference. I will add an issue and put it on the 4.0 milestone

Reply:
done: #4631